client/app/types/course/material/folders.ts

```diff
@@ -12,11 +12,10 @@ export type FolderPermissions = Permissions<
   | 'canCreateSubfolder'
   | 'canUpload'
   | 'canEdit'
-  | 'canEditSubfolders'
 >;
 
 export type SubfolderPermissions = Permissions<
-  'canStudentUpload' | 'showSdlWarning' | 'canEdit' | 'canDelete'
+  'canStudentUpload' | 'showSdlWarning' | 'canEdit' | 'canDelete' | 'canObserve'
```

Comment:
Remove this? It seems that `canObserve` does not exist in the API response in `course/material/folders/show.json.jbuilder`.